### DISCUSSION TITLE
Could not dup tarfh: Bad file descriptor

### DISCUSSION THREAD

**Discussion by andriusst** [ipxe/ipxe#191] [Q&A] [answered]
This is the error message I am getting while trying to build EFI or any iPXE image on rom-o-matic.dev web builder. Just wanted to bring this to system administrator's attention.
Also wanted to say a BIG THANK YOU to people (or person) keeping this project free for the public. I personally don't do frequent builds but when I do I love the flexibility and simplicity and the speed! It offers great convenience for people like myself who don't have a linux machine running with all the dev environment set up and just want a quick fresh build once in a while.

**Comment by NiKiZe** [comment, marked as answer]
You should take that up in the rom-o-matic git repo, unless you can reproduce the issue in a clean build on your own machine.

**Reply by andriusst** [reply]
Doh! excuse my ignorance. I was pointed to this repo and did not realise there is a separate one for the rom-o-matic. Thank you for reply. I will try and post there. My sincere thanks above still apply. You guys are doing a great job! Take care and have a lovely New Year!
